applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,81 @@
+
+This is made possible by SLM's support of CMUX and PPP and Zephyr's generic modem driver.
+
+The :ref:`nRF52840 SoC present in the nRF9160 DK <zephyr:nrf9160dk_nrf52840>` can even be made to be the controlling chip with the appropriate configuration (described below).
+
+.. note::
+
+As of |NCS| v2.6, Zephyr's GNSS driver does not support the nRF91 Series running SLM.
+This functionality is part of planned future work.
+
+Configuration
+*************
+
+Both SLM and the Zephyr application running on the controlling chip must be compiled with the appropriate configuration to make them work together.
+
+For that, Kconfig fragments and Devicetree overlays must be added to the compilation command.
+
+See :ref:`slm_config_files` for a description of some of the mentioned Kconfig fragments.
+
+For the nRF91 Series SiP running SLM
+====================================
+
+The following configuration files must be included:
+
+* :file:`overlay-cmux.conf` - To enable CMUX.
+* :file:`overlay-ppp.conf` - To enable PPP.
+* :file:`overlay-zephyr-modem.conf` - To tailor SLM to how Zephyr's generic modem driver works.
+  This enables :ref:`CONFIG_SLM_START_SLEEP <CONFIG_SLM_START_SLEEP>` which makes the nRF91 Series SiP start only when the :ref:`power pin <CONFIG_SLM_POWER_PIN>` is toggled.
+
+In addition, if the controlling chip is an external MCU:
+
+* :file:`overlay-zephyr-modem-external-mcu.conf`- To define the power pin.
+  Make sure to update its value as needed so that it matches your setup.
+* :file:`overlay-external-mcu.overlay` - To configure the UART to be used.
+  Make sure to update its configuration (pins, baud rate) as needed so that it matches your setup.
+  To update the configuration of the UART, you can directly modify the ``uart2`` node in the :file:`*_ns.overlay` file matching your board in the :file:`boards` directory.
+
+Or if the controlling chip is the nRF52840 of the nRF9160 DK:
+
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.conf` - To define the power pin.
+* :file:`overlay-zephyr-modem-nrf9160dk-nrf52840.overlay` - To configure the UART to be routed between the nRF9160 and the nRF52840 of the DK.
+
+For the controlling chip running Zephyr
+=======================================
+
+It is advised to use the configuration of Zephyr's :ref:`zephyr:cellular_modem` sample as a starting point.
+
+Specifically, the Kconfig options found in the following files:
+
+* :file:`prj.conf` - This file enables various Zephyr APIs, most of which are needed for proper functioning of the application.
+* :file:`boards/nrf9160dk_nrf52840.conf` - This file tailors the configuration of the modem subsystem and driver to the SLM.
+
+In addition, depending on what the controlling chip is, you can use the following Devicetree overlay file as a basis.
+It defines the modem along with its UART and power pin.
+
+If the controlling chip is an external MCU:
```

Comment:
Should this mention something about wiring the uarts together?

Reply:
Could, yeah.